src/ads.pri

```diff
@@ -14,7 +14,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Notepad Next.  If not, see <https://www.gnu.org/licenses/>.
 
-QT += core gui widgets
+QT += core gui widgets gui-private
```

Comment:
I'm not familiar with `gui-private. Does this hurt anything to keep this in for Windows builds? Or should there be a unix block that just adds `gui-private` to the unix builds?

Reply:
Hi @dail8859 it doesn't hurt windows builds. It is required for docking code to compile. https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/blob/6fb45c055c25f838e7cdb9c6934c1f4968b2a1ea/src/ads_globals.cpp#L45 this line caused to issue.